src/shopping-list/shopping-list.css

```diff
@@ -0,0 +1,170 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+* {
+  font-family: 'Roboto Condensed', sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 1200px;
+  margin: auto;
+  background-color: #f5f5f5;
+  color: #4e4e4e;
+}
+
+h3 {
+  margin: 0;
+}
+
+h5 {
+  padding-left: 10px;
+  margin: 1rem 0 0.6rem;
+}
+
+main {
+  margin: 4rem 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  grid-gap: 8rem;
+  padding: 2rem;
+}
+
+.list-ingredients__container {
+  padding: 0.3rem 1.3rem;
+  margin-bottom: 0.3rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 16px;
```

Comment:
if you set font-size in ::root and then in classes you will use rem, you will get more responsive design by default. Please correct me if i am wrong

Reply:
font-sizing units have been changed to rem, as well as the paddings and margins